includes/definitions.inc.php

```diff
@@ -1435,6 +1435,19 @@
 $config['os'][$os]['over'][2]['graph'] = 'device_mempool';
 $config['os'][$os]['over'][2]['text']  = 'Memory Usage';
 
+// Lenovo EMC (NAS)
+$os = 'lenovoemc';
```

Comment:
Does Lenovo give the OS a name?  EMC seems like it could be confusing as well, since that's a well-established SAN/NAS company, and this device has nothing to do with them.